src/node_report.cc

```diff
@@ -1020,6 +1024,11 @@ static void PrintGCStatistics(std::ostream& out, Isolate* isolate) {
  ******************************************************************************/
 static void PrintResourceUsage(std::ostream& out) {
   char buf[64];
+  double cpu_abs;
+  double cpu_percentage;
+  long int diff_time = current_time - load_time;
```

Comment:
difftime() produces result with resolution of seconds, while the time() command gives the result in terms of millis. While in most of the production cases this is immaterial, short-running apps will loose the milli-second differences. Is that fine with you?